pyrasterframes/src/main/python/docs/reference.pymd

```diff
@@ -412,72 +409,64 @@ Performs natural logarithm of cell values plus one. Inverse of @ref:[`rf_expm1`]
 The following functions compute a statistical summary per row of a `tile` column. The statistics are computed across the cells of a single `tile`, within each DataFrame Row.  Consider the following example.
 
 ```python
-import pyspark.functions as F
+import pyspark.sql.functions as F
 spark.sql("""
- SELECT 1 as id, rf_make_ones_tile(5, 5, 'float32') as t 
+ SELECT 1 as id, rf_make_ones_tile(5, 5, 'float32') as t
  UNION
- SELECT 2 as id, rf_local_multiply(rf_tile_ones(5, 5, 'float32'), 3) as t 
+ SELECT 2 as id, rf_local_multiply(rf_make_ones_tile(5, 5, 'float32'), 3) as t
```

Comment:
you may also consider moving this entire example from here to the aggregations chapter. It would make this page more streamlined and help the reader find the content ...